.github/workflows/unit_testing_wf.yaml

```
@@ -0,0 +1,23 @@
+name: macrophage-striation-unittests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  unit-test-core:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment:  macrophage-striation
```

Minor suggestion to remove additional space.
```suggestion
          activate-environment: macrophage-striation
```